quickstart/nodejs/redistest.js

```diff
@@ -1,41 +1,50 @@
 var redis = require("redis");
+require('dotenv').config()
```

Comment:
Is dotenv necessary? We don't have a .env file

Reply:
Sure I'll remove it.

Reply:
Looks like `require('dotenv').config()` is still here